python/pip_install/create_incremental_repo/test_create_incremental_repo.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import argparse
+from tempfile import NamedTemporaryFile
+
+from python.pip_install.create_incremental_repo import generate_incremental_requirements_contents
+from python.pip_install.extract_wheels.lib.bazel import (
+    sanitised_repo_library_label,
+    create_incremental_repo_prefix,
+    sanitised_repo_file_label
+)
+
+
+class TestGenerateRequirementsFileContents(unittest.TestCase):
+
+    def test_incremental_requirements_bzl(self) -> None:
+        with NamedTemporaryFile() as requirements_lock:
+            requirement_string = "foo==0.0.0"
+            requirements_lock.write(bytes(requirement_string, encoding="utf-8"))
+            requirements_lock.flush()
+            args = argparse.Namespace()
+            args.requirements_lock = requirements_lock.name
+            args.repo = "pip_incremental"
+            contents = generate_incremental_requirements_contents(args)
+            library_target = sanitised_repo_library_label("foo", create_incremental_repo_prefix(args.repo))
```

Comment:
in a test I'd prefer you hardcode expected values rather than calculate them, both to document what it looks like, and as an additional assertion that the sanitised_* helper does what we expect